agimus_demo_03_mpc_dummy_traj/dependencies.repos

```diff
@@ -0,0 +1,35 @@
+repositories:
+  vcs_repos/linear_feedback_controller_msgs:
+    type: git
+    url: https://github.com/loco-3d/linear-feedback-controller-msgs
+    version: 0.1.3
+
+  vcs_repos/coal:
+    type: git
+    url: https://github.com/coal-library/coal
+    version: v3.0.0
+
+  vcs_repos/pinocchio:
+    type: git
+    url: https://github.com/stack-of-tasks/pinocchio
+    version: v3.3.0
+
+  vcs_repos/colmpc:
+    type: git
+    url: https://github.com/agimus-project/colmpc
+    version: v0.2.0 # TODO: change to new version tag
+
+  vcs_repos/mim_solvers:
+    type: git
+    url: https://github.com/machines-in-motion/mim_solvers
+    version: v0.0.3
+
+  vcs_repos/franka_ros:
+    type: git
+    url: https://github.com/frankaemika/franka_ros
+    version: develop
```

Comment:
```suggestion
  vcs_repos/franka_ros:
    type: git
    url: https://github.com/frankaemika/franka_ros2
    version: v0.1.15
```
Didn't we end up forking `franka_ros2` in the end?

Reply:
yes here: https://github.com/agimus-project/franka_ros2